docs/blog/20230201-Daeraxa-FebUpdate.md

```diff
@@ -0,0 +1,115 @@
+
+So instead of trying to fix the old one, the current goal is to look at using
+the [WASM version](https://github.com/tree-sitter/tree-sitter/blob/master/lib/binding_web/README.md)
+and [@Maurício Szabo] has been doing some fantastic work on getting this
+implemented into Pulsar. This post isn't for the specifics of this work save to
+say we have seen "interesting" issues and challenges with this but the work is
+looking very promising and if all works out then our main barrier to a major
+Electron upgrade will be out of our way.
+
+## Package repository
+
+There has been an awful lot of work going on in this area and it is mostly
+work that, in theory, should be pretty invisible to most users of Pulsar if all
+goes well.
+
+[@confused-techie] and [@Digitalone1] have been doing some amazing work to improve
+this area with updates to the way it handles versioning and a refactoring of the
+git and github interactions so hopefully we might support other platforms in the
+future.
+
+[@Spiker985] has been creating and updating the [Swagger/OpenAPI definition](https://api.pulsar-edit.dev/swagger-ui/)
+for the backend API so we have a proper definition and validation of it and
+[@confused-techie] started the [Architecture document](https://github.com/pulsar-edit/package-backend/blob/main/ARCHITECTURE.md)
+which explains how everything works at a high level to allow people to more
+easily contribute to the project. We hope to include more of these in other
+repositories as time goes on.
+
+In terms of some package related housekeeping, one of our main goals was to
+preserve as many of the Atom packages as possible. So when we found that some
+packages had some ambiguous or missing licences we have made the effort to
+contact the owner of each package and ask for permission to host them.
+The good news is that this has been well received and we have managed to
+preserve more than a few of them.
+
+## Donations and binary signing
+
+We have had an awful lot of generous donations sent to us recently. In
+particular we have to give a massive shout out to **_@SubAtomic_** who has
+donated more money to the project alone than we thought we get in a whole year.
+
+**_@anonCoffee_** also gave us a very generous donation and was also the first
+sponsor on our new [GitHub Sponsorship](https://github.com/sponsors/pulsar-edit)
+which is an alternative should you want to donate by a platform other than our
+existing [Open Collective](https://opencollective.com/pulsar-edit).
+
+We also now have a webhook set up for the GitHub sponsors so now when we get
+a donation we can properly thank them as it pops up on our Discord `#donations`
+channel.
+
+A huge thank you to literally all of our other donators on both platforms, we
+appreciate everything you give and make so much of what we want to do, possible.
+
+Our first big expenditure we voted for (outside of ongoing hosting costs) will
+be to pay for licences from Apple and Microsoft so that we can finally sign our
```

Comment:
```suggestion
be to pay for Code Signing Certificates from Apple and Microsoft so that we can finally sign our
```

Just updating this to what feels a bit more accurate.

Reply:
Makes sense, exactly what I was after